flask/tests/conftest.py

```diff
@@ -338,6 +338,26 @@ def test_database(client):
     dataset_1.analyses.append(analysis_3)
     dataset_4.analyses.append(analysis_3)
 
+    # Creating a test dataset with tissue samples but no analysis
+    dataset_5 = Dataset(
+        dataset_id=5,
+        dataset_type="WES",
+        condition=DatasetCondition.Somatic,
+        created_by_id=admin.user_id,
+        updated_by_id=admin.user_id,
+    )
+    sample_3.datasets.append(dataset_5)
+
+    # Creating a tissue sample with only one dataset
+    sample_4 = TissueSample(
+        tissue_sample_id=4,
+        tissue_sample_type=TissueSampleType.Blood,
+        created_by_id=admin.user_id,
+        updated_by_id=admin.user_id,
+    )
+    sample_4.datasets.append(dataset_5)
+    participant_3.tissue_samples.append(sample_4)
```

Comment:
Does dataset_5 belong to sample_3 or sample_4?

Reply:
Dataset 5 should belong to sample 4. The sample 3 line there is not really necessary. I will remove.